README.md

```diff
@@ -53,7 +54,9 @@ Install the prerequisites for your desired language:
 
 ### Windows
 
-Both v1 and v2 of the runtime can be installed on Windows. v2 is cross platform, but is still in preview and not recommended for production use. v1 is Windows-only, but is approved for production use.
+Both v1 and v2 of the runtime can be installed on Windows. [See here](https://docs.microsoft.com/azure/azure-functions/functions-versions) for more information on each version.
+
+> **IMPORTANT**: v2 is becoming generally available (GA) soon (as of 9/19/2018). After GA, the "core" tag will no longer be required for v2, and a "v1" tag will be introduced for v1.
```

Comment:
Rather than talking about tags, I wonder if it would be better to be explicitly about the npm install instructions after GA.